controlnext/templates/controlnext/basin_detail.html

```diff
@@ -18,8 +19,10 @@
 <link href="{{ STATIC_URL }}controlnext/widgets/css/quadrant.css" rel="stylesheet"></link>

```

Comment:
Netter: een `{% load staticfiles %}` bovenin de template en dan een link zoals dit:

    <link href="{% static 'bootstrap/dist/css/bootstrap.min.css' %}" rel="stylesheet">

Dan kan staticfiles netjes gehashte versies van de css maken in plaats van dat er alleen maar een STATIC_URL voor geplakt wordt.